pkgs/development/python-modules/py7zr/default.nix

```diff
@@ -0,0 +1,60 @@
+{
+  brotli,
+  brotlicffi,
+  buildPythonPackage,
+  fetchPypi,
+  inflate64,
+  lib,
+  multivolumefile,
+  nix-update-script,
+  psutil,
+  pybcj,
+  pycryptodomex,
+  pyppmd,
+  pythonOlder,
+  pyzstd,
+  setuptools,
+  setuptools-scm,
+  texttable,
+}:
+
+buildPythonPackage rec {
+  pname = "py7zr";
+  version = "0.22.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-xseupZE1NRhAA7c5OEkPmk2EGFmOUz+cqZHTuORaE54=";
+  };
+
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
+  dependencies = [
+    brotli
+    brotlicffi
+    inflate64
+    multivolumefile
+    psutil
+    pybcj
+    pycryptodomex
+    pyppmd
+    pyzstd
+    texttable
+  ];
+
```

Comment:
no tests?

Reply:
I've enabled for tests for all of the packages that have them. Most of them have tests available. The unrar2-cffi tests aren't passing, unfortunately.